nucliadb_protos/writer.proto

```diff
@@ -83,7 +83,6 @@ message BrokerMessage {
 
     resources.Origin origin = 9;
 
-    repeated utils.Relation relations = 10;
```

Comment:
can you add the 10 to the reserved field numbers avobe?

Reply:
Done. I also added a migration to remove the maindb keys that are no longer needed.